jsk_pcl_ros/scripts/install_sample_data.py

```diff
@@ -139,6 +139,11 @@ def download_data(**kwargs):
         md5='dddd2c6d7738ac19536a032aed8f9967',
     )
 
+    download_data(
+        path='sample/data/baxter_realsense_l515.bag',
+        url='https://drive.google.com/uc?id=1RKZLA4J2_lzhCtvMTWxGQh4ykVVEeSAd',
+        md5='4dab24bd6e6a7fd5b9e86d053352cdfd',
+    )
```

Comment:
@knorth55 did you update the `baxter_realsense_l515.bag` file recently?  It seems the file is updated on 11/17 
https://drive.google.com/drive/u/1/search?q=baxter_realsense_l515.bag

![Screenshot from 2022-11-24 17-05-52](https://user-images.githubusercontent.com/493276/203727226-4dab13ee-3af3-4805-bcd3-5955dcf4a902.png)

Reply:
I'm sorry, I update the rosbag file for this PR.

Reply:
not for this PR, for this PR https://github.com/jsk-ros-pkg/jsk_recognition/pull/2656

Reply:
I made a PR to fix this issue https://github.com/jsk-ros-pkg/jsk_recognition/pull/2751